ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerFile/main.qml

```diff
@@ -0,0 +1,12 @@
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.Samples 1.0
+
+ApplicationWindow {
+    visible: true
+    width: 800
+    height: 600
+
+    RasterLayerFile {
```

Comment:
I wonder if it would be better to set the anchors here - rather than hard coding in the sample type?